src/hooks/useRepo.ts

```diff
@@ -0,0 +1,14 @@
+import useSWR from "swr";
+
+const useRepo = (name: string) => {
+  const { data, error, mutate } = useSWR<DbRepo, Error>(`repos/${name}`);
+
```

Comment:
What happens when if you get an error? You have to treat this issue. Or to not break this PR you can open one issue with this issue.

Reply:
please stop commenting if you are not testing - this is handled via swr, don't have to do anything unless you want to change the error message; there's an outstanding issue opened by you about the toaster 🍕